react-native/react/local-debug.desktop.js

```diff
@@ -18,8 +18,8 @@ let config = {
   reduxDevToolsEnable: false,
   redirectOnLogout: true,
   reduxDevToolsSelect: state => state, // only watch a subset of the store
-  enableStoreLogging: false,
-  forwardLogs: false
+  enableStoreLogging: true,
+  forwardLogs: true
```

Comment:
Hmm that's a good point, is there anything we can do to make it faster or not block?